ci/run_integration_test.sh

```diff
@@ -1,6 +1,6 @@
 #! /bin/bash -e
 
-BRANCH='master'
+BRANCH='4.6'
```

Comment:
The latest folder does not contain all the Scylla files.
I think it better to choose 4.5